content/en/getting_started/logs/_index.md

```diff
@@ -0,0 +1,223 @@
+
+* [Create a Datadog account](#create-a-datadog-account)
+* [Send Logs manually](#sending-logs-manually)
+* [Use the Agent to send logs from a file](#send-logs-from-a-file)
+
+Before moving forward:
+
+1. If you haven't already, create a [Datadog account][1] and [enable Datadog Log management][2].
+2. Set up a [Vagrant Ubuntu 16.04 virtual machine][3] using the following commands. For more information about Vagrant, see their [Getting Started][4] page:
+
+    ```
+    vagrant init ubuntu/xenial64
+    vagrant up
+    vagrant ssh
+    ```
+
+## Sending logs Manually
+
+To send logs manually, use the `telnet` command with your [Datadog API key][5] within the vagrant virtual machine.
+
+Logs can be a full-text message:
+
+{{< tabs >}}
+{{% tab "US Site" %}}
+
+The secure TCP endpoint is `intake.logs.datadoghq.com:10516` (or port `10514` for insecure connections).
+
+```
+telnet intake.logs.datadoghq.com 10514
+
+<DATADOG_API_KEY> Plain text log sent through TCP
+```
+
+{{% /tab %}}
+{{% tab "EU Site" %}}
+
+The secure TCP endpoint is `tcp-intake.logs.datadoghq.eu:443` (or port `1883` for insecure connections).
+
+```
+telnet tcp-intake.logs.datadoghq.eu 1883
+
+<DATADOG_API_KEY> Plain text log sent through TCP
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+This produces the following result in your [Log Explorer Page][2]:
+
+{{< img src="getting_started/logs/plain_text_log.png" alt="Custom telnet" responsive="true">}}
+
+or a JSON object that is automatically parsed by Datadog:
+
+{{< tabs >}}
+{{% tab "US Site" %}}
+
+```
+telnet intake.logs.datadoghq.com 10514
+
+<DATADOG_API_KEY> {"message":"JSON formatted log sent through TCP", "ddtags":"env:dev", "ddsource":"terminal", "hostname":"gs-hostame", "service":"user"}
+```
+
+{{% /tab %}}
+{{% tab "EU Site" %}}
+
+```
+telnet tcp-intake.logs.datadoghq.eu 1883
+
+<DATADOG_API_KEY> {"message":"JSON formatted log sent through TCP", "ddtags":"env:dev", "ddsource":"terminal", "hostname":"gs-hostame", "service":"user"}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+This produces the following result in your [Log Explorer Page][2]:
+
+{{< img src="getting_started/logs/json_log.png" alt="JSON logs" responsive="true">}}
+
+## Send logs from a file
+
+### Install the Agent
+
+To install the Datadog Agent within your Vagrant host, use the [one line install command][6] updated with your [Datadog API key][7]:
+
+{{< tabs >}}
+{{% tab "US Site" %}}
+
+```
+DD_API_KEY=<YOUR_DD_API_KEY>  bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
+```
+
+{{% /tab %}}
+{{% tab "EU Site" %}}
+
+```
+DD_API_KEY=<YOUR_DD_API_KEY> DD_SITE="datadoghq.eu" bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Validation
+
+Verify the Agent is running with the [status command][8]:
+
+```
+sudo datadog-agent status
+```
+
```

Comment:
Should we show the example of what to expect? (with the line `logs collection not started` for instance)?

Reply:
🤔 we could indeed. i'm going to add it.